config/routes.rb

```diff
@@ -13,20 +13,38 @@
   end
 
   namespace :editor do
-    root 'welcome#index'
+    get "/(:controller_name)/(:id)/(:action)", to: redirect {|params, request|
+      base_url = "#{request.scheme}://#{request.host_with_port}/sites/#{request.host}/editor"
+      path = [params[:controller_name], params[:id], params[:action]].compact.join("/")
+      if path.empty?
+        base_url
+      else
+        "#{base_url}/#{path}"
+      end
+    }
+  end
+
+  namespace :sites do
```

Comment:
I've implemented this according to https://github.com/bm-sms/daimon-news-misc/issues/136#issue-160084273

When we use `namespace :editor`, I think it is difficult to impliment redirection.
Maybe `resource :sites do...` is ok.